crates/dbsp/benches/gdelt/data.rs

```diff
@@ -302,7 +303,9 @@ pub fn build_gdelt_normalizations() -> (Interner, Normalizations, Invalid) {
         interner.extend(NORMALS.iter().flat_map(|&(_, person)| person).cloned());
 
         let mut map = HashMap::with_capacity_and_hasher(NORMALS.len(), Xxh3Builder::new());
-        map.extend(NORMALS);
```

Comment:
I think you can just use `map.extend(NORMALS.iter().copied())`

Reply:
That's right, thanks! I pushed an update.